py4cast/datasets/poesy.py

```diff
@@ -257,6 +271,62 @@ def num_total_steps(self) -> int:
         return self.num_input_steps + self.num_output_steps
 
 
+#############################################################
+#                      BROWSE DATASET                       #
+#############################################################
+
+
+def browse_poesy(period: Period, settings: PoesySettings):
+    """
+    Create a list of arguments used to instantiate Sample. 
+    This function indicates how to run through / browse the Poesy.    
+    """
+    
+    list_args_all_samples = []
+
+    # Create all datetimes
+    date_list = pd.date_range(
```

Comment:
following a remark of @cbovalo in previous pr, no need for pandas here, maybe just use

`return np.arange(
            start, end, np.timedelta64(step, "h"), dtype="datetime64[s]"
        ).tolist()`